tools/bump_version.sh

```diff
@@ -34,42 +88,33 @@ do
 
         shift 2
         ;;
-    *)
-        if [[ $1 =~ ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-[[:alnum:]]+)?$ ]]
+    "-d")
+        if [[ $2 =~ ^[0-9]{4}-((0[1-9])|(1[0-2]))-(([0-2][1-9])|(3[01]))$ ]]
```

Comment:
The following change is needed for days: `10` and `20`.
```suggestion
        if [[ $2 =~ ^[0-9]{4}-((0[1-9])|(1[0-2]))-(([0-2][0-9])|(3[01]))$ ]]
```

Reply:
Fixed! Thanks!